packages/ui/src/hooks/useBalances.ts

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react'
+import { AccountInfo } from '@polkadot/types/interfaces'
+import { Account } from './useAccounts'
+import { useApi } from './useApi'
+
+interface Balance {
+  total: string
+}
+
+interface AddressToBalanceMap {
+  [key: string]: Balance
```

Comment:
How about introducing `type Address = string`? Could be used here and in the `Account` type as well.